### PR TITLE
Disabling and Enabling the sync plan in Capsule to avoid Bugzilla 1721055 

### PR DESCRIPTION
**Pull request by devendra104**
#328

### CODE REVIEW

File: upgrade/helpers/tasks.py [hunk 1]
```diff
@@ -95,6 +95,9 @@ def sync_capsule_repos_to_upgrade(capsules):
     RHEV_CAPSULE_AK
         The AK name used in capsule subscription
     """
+    setup_foreman_maintain()
+    logger.info('Disabling the sync plan ...')
```
Comment (jyejare) [inline upgrade/helpers/tasks.py]: Disabling untill ``` bug ID  ``` fixes will be helpfull.

File: upgrade/helpers/tasks.py [hunk 2]
```diff
@@ -95,6 +95,9 @@ def sync_capsule_repos_to_upgrade(capsules):
...
+    run('foreman-maintain advanced procedure run sync-plans-disable')
```
Comment (jyejare) [inline upgrade/helpers/tasks.py]: use ```skip_if_bug_open``` decorator.

File: upgrade/helpers/tasks.py [hunk 3]
```diff
@@ -132,6 +135,8 @@ def sync_capsule_repos_to_upgrade(capsules):
         if capsuletools_url:
             add_custom_product_subscription_to_hosts(
                 customcontents['capsule_tools']['prod'], capsules)
+    logger.info("Enabling the sync plan...")
+    run('foreman-maintain advanced procedure run sync-plans-enable')
```
Comment (jyejare) [inline upgrade/helpers/tasks.py]: same as above

File: upgrade/helpers/tasks.py (hunk 1 above)
Comment (devendra104) [reply upgrade/helpers/tasks.py]: Same has been incorporated.

File: upgrade/helpers/tasks.py (hunk 2 above)
Comment (devendra104) [reply upgrade/helpers/tasks.py]: Same has been incorporated.

File: upgrade/helpers/tasks.py [hunk 4]
```diff
@@ -98,6 +98,10 @@ def sync_capsule_repos_to_upgrade(capsules):
     command = "foreman-maintain health check --label " \
               "foreman-tasks-not-running -y"
     check_status_of_running_task(command, 3)
+    setup_foreman_maintain()
```
Comment (jyejare) [inline upgrade/helpers/tasks.py]: can we also add this under bz_bug ....

Comment (devendra104) [reply upgrade/helpers/tasks.py]: Fixed.

**Pull request merged**